pages/index.js

```diff
@@ -1,11 +1,25 @@
-import Link from "next/link";
+import {
+  StyledAddTripButtonOverview,
+  StyledHeaderOverview,
+  StyledHeaderRightSideOverview,
+  Styledtitle,
+} from "@/components/TripList/TripList.styled";
+
 import TripList from "@/components/TripList";
 
 export default function HomePage() {
   return (
     <main>
-      <h1>My Trips</h1>
-      <Link href="/addTrip">+ Add Trip</Link>
+      <StyledHeaderOverview>
+        <Styledtitle>
+          <h1>My Trips</h1>
```

Comment:
The Styledtitle should not be styled.div, instead, you should use styled.h1. Then you can delete the h1 tags in line 15 🥸

Reply:
danke! hab ich erledigt!